ERCS/erc-7662.md

```diff
@@ -0,0 +1,273 @@
+---
+eip: 7662
+title: AI Agent NFTs
+description: A specification for NFTs that represent AI Agents.
```

Comment:
Your description doesn't include any new information that wasn't in your title, and it has a bit of fluff ("A specification for".) You should use the description to elaborate on the ideas introduced in the title as succinctly as possible.

For example, what is an AI Agent?

Reply:
Good point - will add, thanks.